### ISSUE TITLE
Add current activity modifiers to characterActivities

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#605]
As the title states, this is a feature request to return an `activeModifiers` property for each entry in `availableActivities` and the current activity when using 204 on GetProfile.

```javascript
{
...
activeModifiers: [
0,
4
];
...
}
```

**Comment by floatingatoll** [comment]
Thorn, #473 could probably be merged here somehow.

**Comment by lowlines** [comment]
Yeah there are a lot of similar requests, though characterActivities is character specific as activities like the Whisper quest don't show up as a milestone yet have a weekly modifier rotation AND a conditional modifier based on the progress on the Taken Ship pursuit.

**Comment by vthornheart-bng** [comment]
Yeah, I dig it - I think that'd be a great addition to the API!
